content/patterns/button/button-pattern.html

```diff
@@ -14,93 +14,94 @@
+            そのような場合、要素に <code>button</code> ロールを与えることは、支援技術利用者が要素の機能を理解するのに役立ちます。
+            ただし、より良い解決策は、機能と ARIA ロールに一致するように視覚デザインを調整することです。
           </p>
         </div>
       </section>
 
       <section id="examples">
         <img alt="" src="../../images/pattern-button.svg">
-        <h2>Examples</h2>
+        <h2>例</h2>
         <ul>
-          <li><a href="examples/button.html">Button Examples</a>: Examples of clickable HTML <code>div</code> and <code>span</code> elements made into accessible command and toggle buttons.</li>
+          <li><a href="examples/button.html">ボタンの例</a>: アクセシブルなコマンドボタン及びトグルボタンにされたクリック可能な HTML <code>div</code> 及び <code>span</code> 要素の例。</li>
           <li>
-            <a href="examples/button_idl.html">Button Examples (IDL)</a>: Examples of clickable HTML <code>div</code> and <code>span</code> elements made into accessible command and toggle buttons.
-            This example uses the <a class="specref" href="#idl-interface">IDL Interface</a>.
+            <a href="examples/button_idl.html">ボタンの例（IDL バージョン）</a>: アクセシブルなコマンドボタン及びトグルボタンにされたクリック可能な HTML <code>div</code> 及び <code>span</code> 要素の例。
+            この例では、<a class="specref" href="#idl-interface">IDL インターフェース</a>を使用しています。
           </li>
         </ul>
       </section>
 
       <section id="keyboard_interaction">
-        <h2>Keyboard Interaction</h2>
-        <p>When the button has focus:</p>
+        <h2>キーボード操作</h2>
+        <p>ボタンにフォーカスがある場合:</p>
         <ul>
-          <li><kbd>Space</kbd>: Activates the button.</li>
-          <li><kbd>Enter</kbd>: Activates the button.</li>
+          <li><kbd>Space</kbd>: ボタンを作動する。</li>
+          <li><kbd>Enter</kbd>: ボタンを作動する。</li>
           <li>
-            Following button activation, focus is set depending on the type of action the button performs. For example:
+            ボタンの作動後、フォーカスはボタンが実行するアクションの種類に応じて設定されます。例:
             <ul>
               <li>
-                If activating the button opens a dialog, the focus moves inside the dialog.
-                (see <a href="../dialog-modal/dialog-modal-pattern.html">dialog pattern</a>)
+                ボタンをアクティブにするとダイアログが開く場合、フォーカスはダイアログ内に移動します。
+                （<a href="../dialog-modal/dialog-modal-pattern.html">ダイアログパターン</a>を参照）
               </li>
               <li>
-                If activating the button closes a dialog, focus typically returns to the button that opened the dialog unless the function performed in the dialog context logically leads to a different element.
-                For example, activating a cancel button in a dialog returns focus to the button that opened the dialog.
-                However, if the dialog were confirming the action of deleting the page from which it was opened, the focus would logically move to a new context.
+                ボタンをアクティブにするとダイアログが閉じる場合、ダイアログのコンテキストで実行された機能が論理的に異なる要素につながる場合を除き、フォーカスは通常、ダイアログを開いたボタンに戻ります。
+                たとえば、ダイアログでキャンセルボタンをアクティブにすると、フォーカスはダイアログを開いたボタンに戻ります。
```

Comment:
activatingを直訳するのは日本語ではわかりづらそう。
「アクティブにする」はもちろん「有効化する」もわかりづらい。
~~「操作する」くらいがいいか？
用語集でもactivity:操作の対訳になっている~~

作動にする
```suggestion
                ボタンを作動するとダイアログが閉じる場合、ダイアログのコンテキストで実行された機能が論理的に異なる要素につながる場合を除き、フォーカスは通常、ダイアログを開いたボタンに戻ります。
                たとえば、ダイアログでキャンセルボタンを作動すると、フォーカスはダイアログを開いたボタンに戻ります。
```